demo.lua

```diff
@@ -58,7 +52,7 @@ local function readfrom(path)
 	return contents
 end
 
-local net = require "cujo.net"
+local net = require "rabid.net"
```

Comment:
Let's keep `cujo` namespace since this module will not be embedded in Rabid anymore, but distributed as an independent module.
Just keep the license change.

Reply:
won't be a bit weird to import `cujo*` from openrabid? should I move the import elsewhere? is that possible?